server.js

```diff
@@ -1,11 +1,29 @@
 var express = require('express');
+var fs = require('fs');
+var bodyParser = require('body-parser');
 
 var app = express();
+var stringifyFile;
```

Comment:
in production mode puting this variable to global scope is not good idea ;) try to write the same using this variable as internal variable of function ;)